miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1072,10 +1011,7 @@ public static SequencingParameters to(SequencingParametersDto from) {
   }
 
   public static List<SequencingParametersDto> asSequencingParametersDtos(Collection<SequencingParameters> from) {
-    List<SequencingParametersDto> dtoList = new ArrayList<>();
-    for (SequencingParameters sp : from) {
-      dtoList.add(asDto(sp));
-    }
+    List<SequencingParametersDto> dtoList = from.stream().map(Dtos::asDto).collect(Collectors.toList());
     Collections.sort(dtoList, new Comparator<SequencingParametersDto>() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)